RELEASE_NOTES.md

```diff
@@ -37,6 +37,60 @@ You'll need to add `tailwind.config.js` to your app `.gitignore`. If you generat
 
 You can read more about this change on PR [\#9480](https://github.com/decidim/decidim/pull/9480).
 
+### 3.2 Change Webpacker to Shakapacker
+
+Since the Rails team has retired the Webpacker in favour or importmap-rails or js-bundling, we got ouserlves in a situation where performance improvements could not be performed.
+In order to continue having support for Webpacker like syntax, we have switched to Shakapacker.
+
+In order to perform the update, you will need to make sure that you do not have webpacker in your Gemfile.
+If you have it, please remove it, or change it to
+
+```ruby
+gem "shakapacker", "~> 6.5"
+```
+
+Please backup the following files, to make sure that you save any customizations you may have done to webpacker:
+
+```console
+config/webpacker.yml
+config/webpack/*
+package.json
+postcss.config.js
+```
+
+After you have performed the backup, make sure you run:
+
+```console
+bundle update decidim
```

Comment:
These commands should be part of the normal upgrade process as they are usually added at the top of the release notes.

We should not require the user to duplicate any steps in the upgrade process.